queue/util/createConsumer.js

```diff
@@ -38,9 +38,18 @@ module.exports = async function (options) {
   }, consumerOptions))
 
   consumer.on('message', async (message) => {
+    let content
+
     try {
-      const content = messageParser.decode(message.content)
+      content = messageParser.decode(message.content)
```

Comment:
Additional change. We should not retry handling the message if it fails to decode because it will never decode properly.